java/algorithm/src/main/java/org/apache/arrow/algorithm/sort/DefaultVectorComparators.java

```diff
@@ -230,6 +235,50 @@ public int compareNotNull(int index1, int index2) {
     }
   }
 
+  /**
+   * Default comparator for {@link BaseRepeatedValueVector}.
```

Comment:
can you document how it compares elements?

Reply:
Good suggestion. Thanks.